src/cli/javascript/request_filter.rs

```diff
@@ -20,6 +21,34 @@ pub enum Command {
     Response(JsResponse),
 }
 
+impl<'js> FromJs<'js> for Command {
```

Comment:
Isn't there a macro to derive FromJS and ToJS?

Reply:
Worth asking on the repo. Wonder how the aws llrt team is doing this.

Reply:
There's a macro for exposing classes that also adjusts prototype chains for you, but not for pure simple objects. llrt uses both, the [macro](https://github.com/awslabs/llrt/blob/63e705b023e00312c7b295fae1ed3625b0e0205d/src/events.rs#L382) and [manual trait implementations](https://github.com/awslabs/llrt/blob/63e705b023e00312c7b295fae1ed3625b0e0205d/src/buffer.rs#L23).